src/xcode/ENA/ENA/Source/Services/CoronaTestService/CoronaTestService.swift

```diff
@@ -183,6 +187,9 @@ class CoronaTestService {
 					Analytics.collect(.testResultMetadata(.updateTestResult(.positive, registrationToken, .pcr)))
 					Analytics.collect(.keySubmissionMetadata(.submittedWithTeletan(true, .pcr)))
 					
+					// updating badge count for home tab
+					self?.unseenTestsCount.value += 1
+
```

Comment:
There is no way currently to add a teletan from the universal scanner so I think it should not be required to mark it as unseen here.